content/en/docs/_index.md

```diff
@@ -4,28 +4,9 @@ linkTitle: Docs
 menu: { main: { weight: 10 } }
 ---
 
-OpenTelemetry is a set of APIs, SDKs, tooling and integrations that are designed
-for the creation and management of _telemetry data_ such as traces, metrics, and
-logs. The OpenTelemetry documentation is intended to broadly cover key terms,
-concepts, and instructions on how to use OpenTelemetry in your software.
+OpenTelemetry, also known as OTel for short, is a vendor-neutral open-source [Observability](concepts/observability-primer/#what-is-observability) framework for instrumenting, generating, collecting, and exporting telemetry data ([traces](concepts/observability-primer/#distributed-traces), [metrics](concepts/observability-primer/#reliability--metrics), [logs](concepts/observability-primer/#logs)). As an industry-standard it is [natively supported by a number of vendors](/vendors)
 
-![Implementation of the OpenTelemetry Reference Architecture](https://raw.github.com/open-telemetry/opentelemetry.io/main/iconography/Reference_Architecture.svg)
+![OpenTelemetry Reference Architecture](/img/otel_diagram.webp)
```

Comment:
This might just be temporary, but in case it isn't, I'd like to flag that webp isn't as widely usable for iOS and macOS users. We should consider using another format (other than .webp and .tiff).

(For the .webp fineprint see, https://caniuse.com/?search=webp.)

Reply:
Noted - I will convert those ASAP so that it doesn't fall off my radar.